liiatools/datasets/cin_census/lds_cin_clean/file_creator.py

```diff
@@ -15,9 +14,8 @@ def convert_to_dataframe(data):
 
 def get_year(input, data, la_log_dir):
     filename = Path(input).stem
```

Comment:
Happy to remove that step, I think it was there to minimise catching any other numbers, for instance these might be nested in a folder called 2022_LIIA/LA/etc...

It would need a bit of tweaking but could maybe fit into common. Just that in this case it's working on a dataframe whereas it's working on a stream for the 903